src/api/app/models/package.rb

```diff
@@ -120,6 +116,23 @@ def self.check_access?(dbpkg = self)
     return Project.check_access?(dbpkg.project)
   end
 
+  def self.dirty_backend_packages
+    # avoid scope here, it will turn objects readonly
```

Comment:
I have the feeling we are talking about 2 different things. I'm talking about activerecord scopes. What do you talk about? :-)